src/core/lib/event_engine/posix_engine/ev_poll_posix.cc

```diff
@@ -559,15 +560,13 @@ uint32_t PollEventHandle::BeginPollLocked(uint32_t read_mask,
   return mask;
 }
 
-EventEngine::Closure* PollEventHandle::EndPollLocked(int got_read,
-                                                     int got_write) {
-  EventEngine::Closure* closure = nullptr;
+bool PollEventHandle::EndPollLocked(int got_read, int got_write) {
```

Comment:
bool got_read/bool got_write? (let's get this bit fixed... it's been a while since we were c89!)

Reply:
Oops fixed it now.